packages/core/src/shared/filetypes.ts

```diff
@@ -355,7 +355,7 @@ export const wellKnownCodeFiles = new Set(['Dockerfile', 'Dockerfile.build'])
 export function isCodeFile(filename: string): boolean {
```

Comment:
please rename `wellKnownCodeFiles` to something like `codefileBaseNames`, so that it follows the existing pattern of `codefileExtensions`.

Reply:
This already exists here and in the JetBrains extension: https://github.com/aws/aws-toolkit-jetbrains/blob/07ccd110d741c4bf591bfcbc5db86a2a7022313f/plugins/core/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonq/FeatureDevSessionContext.kt#L72

Reply:
Please rename it.

Reply:
Okay, done.